test/convenience/inline_query.test.ts

```diff
@@ -0,0 +1,2322 @@
+                { caption: "cap" },
+            );
+            assertObjectMatch(audio, {
+                type: "audio",
+                id: "id",
+                title: "title",
+                audio_url: "https://grammy.dev/",
+                caption: "cap",
+            });
+        });
+        it("should build an InlineQueryResultAudio from a URL", () => {
+            const audio = InlineQueryResultBuilder.audio(
+                "id",
+                "title",
+                new URL("https://grammy.dev/"),
+                { caption: "cap" },
+            );
+            assertObjectMatch(audio, {
+                type: "audio",
+                id: "id",
+                title: "title",
+                audio_url: "https://grammy.dev/",
+                caption: "cap",
+            });
+        });
+
+        describe("with text", () => {
+            it("should build an InlineQueryResultAudio with location in message content", () => {
+                const audio = InlineQueryResultBuilder.audio(
+                    "id",
+                    "title",
+                    "https://grammy.dev/",
+                    { caption: "cap" },
+                ).text("#Text", { parse_mode: "Markdown" });
+
+                assertObjectMatch(audio, {
+                    type: "audio",
+                    id: "id",
+                    title: "title",
+                    audio_url: "https://grammy.dev/",
+                    caption: "cap",
+                    input_message_content: {
+                        message_text: "#Text",
+                        parse_mode: "Markdown",
+                    },
+                });
+            });
+        });
+
+        describe("with location", () => {
+            it("should build an InlineQueryResultAudio with location in message content", () => {
+                const audio = InlineQueryResultBuilder.audio(
+                    "id",
+                    "title",
+                    "https://grammy.dev/",
+                    { caption: "cap" },
+                ).location(83, 136, { heading: 1 });
+
+                assertObjectMatch(audio, {
+                    type: "audio",
+                    id: "id",
+                    title: "title",
+                    audio_url: "https://grammy.dev/",
+                    caption: "cap",
+                    input_message_content: {
+                        latitude: 83,
+                        longitude: 136,
+                        heading: 1,
+                    },
+                });
+            });
+        });
+
+        describe("with venue", () => {
+            it("should build an InlineQueryResultAudio with venue in message content", () => {
+                const audio = InlineQueryResultBuilder.audio(
+                    "id",
+                    "title",
+                    "https://grammy.dev/",
+                    { caption: "cap" },
+                ).venue("Grammy Venue", 83, 136, "1 Grammy Venue", {
```

Comment:
Sorry for being pedantic about this one, please correct it in all other places 🤓 
```suggestion
                ).venue("grammY Venue", 83, 136, "1 grammY Venue", {
```